examples/pascal_voc_jaccard_loss.py

```diff
@@ -0,0 +1,84 @@
+'''
+Trains a DenseNet-40-12 model on the CIFAR-10 Dataset.
```

Comment:
shouldn't this file be called something like cifar10_jaccard_loss.py?

Reply:
Ops I need to change that docstring, it has to use pascal voc because it's showing how jaccard loss is good for segmention.